### DISCUSSION TITLE
Should we release 0.70.6?

### DISCUSSION THREAD

**Discussion by cipolleschi** [reactwg/react-native-releases#44] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/48

---

Current Release: [0.70.5](https://github.com/facebook/react-native/releases/tag/v0.70.5)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/0e316ec671617f5e7c1985b4b05cd0d45bcea403
1. https://github.com/facebook/react-native/commit/475310dbbaec8048411edefc6cdddab330df7966
1. Bump CLI to https://github.com/react-native-community/cli/releases/tag/v9.3.2
1. remove "@jest/create-cache-key-function": "^29.0.3", from package.json, no idea how it ended up in the cherry pick (https://github.com/facebook/react-native/commit/f5ebb692ab9b3ca40fb1dc79c5902f04d04f7a55) when it wasn't in the PR (https://github.com/facebook/react-native/pull/35022/files)
    1. https://github.com/facebook/react-native/pull/35014 (It's used in the jest preset)  

#### Local commits to backport to main

1.

**Comment by thymikee** [comment]
Request to pick latest CLI v9, currently: https://github.com/react-native-community/cli/releases/tag/v9.3.2

**Comment by glenlindquist** [comment]
Request to pick https://github.com/facebook/react-native/commit/9ac437f25b51b9a7fc8134d17c69b64ae90d508d 
Add support for Modal on iOS when Fabric is enabled.

**Reply by cipolleschi** [reply]
Hi @glenlindquist, thank for your request, but we will not cherry pick this commit because it is an addition and we usually only pick fixes for patch releases. 
You can read more about that [here](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request).
